app/javascript/packs/stylesheets/analytics.scss

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
Good idea breaking this out into its own file

Reply:
this was actually something Adam had asked me to do haha

https://github.com/SaraAlert/SaraAlert/pull/239#discussion_r455381974